tools/freebayes/freebayes.xml

```diff
@@ -1,4 +1,4 @@
-<tool id="freebayes" name="FreeBayes" version="1.0.2.29">
+<tool id="freebayes" name="FreeBayes" version="1.0.2.29_galaxy1">
```

Comment:
I'm fine with whatever the group decides, though I don't understand "Galaxy independent tool description". This version specifically refers to the Galaxy wrapper.